Translation/en/RedirectedResources/assets/abdata/adv/scenario/c03/penetration/masturbation_answer/translation.txt

```diff
@@ -1,5 +1,5 @@
 //
 // Dumped for Koikatsu Sunshine v1.0.1 (KoikatsuSunshine) by KKS_TextDump v1.4.5.0
 //
-//「ま、まさかっ……エッチしたいなんて……」=
-//「も、もちろん……いいですよ……？　その、い、今こんなことしてたですから……私だって、準備万端……なのです」=
+「ま、まさかっ……エッチしたいなんて……」=“N-no way ...... you want to have sex ......”
+「も、もちろん……いいですよ……？　その、い、今こんなことしてたですから……私だって、準備万端……なのです」=“O-of course ...... it's fine ......? Well, s-since I was just doing this ...... I'm ready ...... as well” 
```

Comment:
```suggestion
「も、もちろん……いいですよ……？　その、い、今こんなことしてたですから……私だって、準備万端……なのです」=“O-of course... it's fine...? Well, s-since I was just doing this... I'm ready... as well” 
```